### PR TITLE
MEN-7672: Update information regarding "out of the box" Update Modules

### PR DESCRIPTION
**Pull request by lluiscampos**
Missing from commit 1cae3d9221e4b38eee5b1410f4ee453de2abd0c9

### CODE REVIEW

File: 05.Operating-System-updates-Yocto-Project/05.Customize-Mender/docs.md [hunk 1]
```diff
@@ -163,13 +163,12 @@ install your own custom Update Modules with Yocto.
 
 ### Standard Update Modules
 
-Mender comes with some Update Modules available out of the box. These are:
+Mender comes with one Update Module to perform full rootfs updates:
+* rootfs-image
```
Comment (lluiscampos) [inline 05.Operating-System-updates-Yocto-Project/05.Customize-Mender/docs.md:167]: Note to self,

Check that it is the case that this module is installed always :memo:

File: 05.Operating-System-updates-Yocto-Project/05.Customize-Mender/docs.md [hunk 2]
```diff
@@ -163,13 +163,12 @@ install your own custom Update Modules with Yocto.
...
+
+Additionally, there are some Update Modules available out of the box. These are:
```
Comment (estenberg) [inline 05.Operating-System-updates-Yocto-Project/05.Customize-Mender/docs.md:169]: Would say

In addition, these Update Modules available for install:

* directory
* single-file

However, they are not enabled by default...

Comment (estenberg) [reply 05.Operating-System-updates-Yocto-Project/05.Customize-Mender/docs.md:169]: It just wasn't clear what "out of the box" means in this context.

I interpreted it as "rootfs-image" is always installed (that I would call out of the box, but don't have to phrase it like that) while the others can easily be installed.

**Review by estenberg — CHANGES_REQUESTED** [review]
Would reword a bit if I interpreted correctly

File: 05.Operating-System-updates-Yocto-Project/05.Customize-Mender/docs.md (hunk 1 above)
Comment (lluiscampos) [reply 05.Operating-System-updates-Yocto-Project/05.Customize-Mender/docs.md:167]: Reply to self,

This module is not treated different than the rest, all are opt-out

File: 05.Operating-System-updates-Yocto-Project/05.Customize-Mender/docs.md (hunk 2 above)
Comment (lluiscampos) [reply 05.Operating-System-updates-Yocto-Project/05.Customize-Mender/docs.md:169]: The "out of the box" was already there. But I will remove it now :+1:

**Comment by lluiscampos** [comment]
@estenberg I investigated further to realize that the documentation was wrong: these modules are installed by default. Please see the rework now (and the rewording of "out of the box" :wink: )

**Comment by lluiscampos** [comment]
Pinging also @TheMeaningfulEngineer and @danielskinstad for a sanity check on the yocto syntax :monocle_face:

**Review by estenberg — APPROVED** [review]
Nice and clear!
